src/js/components/stock-movement-wizard/outboundImport/OutboundImport.jsx

```diff
@@ -78,15 +80,27 @@ const OutboundImport = () => {
     trigger,
   };
 
+  /**
+   * Skips validation to allow form submission after a page refresh,
+   * where the required XLS file is lost. The file was already validated earlier,
+   * so we bypass validation and submit the form directly to retain progress.
+   *
+   * Related ticket OBPIH-6627 (Keep filled form progress when refreshing the page)
+   */
+  const handleConfirmSubmitForm = (submitMethod) => (event) => {
+    event.preventDefault();
+    submitMethod(getValues());
+  };
+
```

Comment:
I kinda do not get this method. Why do we want to skip validation? And which validation exactly? Is it the frontend validation of the file from previous step? Even if so, how do you distinguish here whether the submit happens from the refreshed page or from regular usage of the workflow?

Reply:
The problem is that I don't cache the uploaded file `packingList` which is a required value in the validation schema,
So after loading all of the values from `sessionStorage` I am missing the `packingList` File which prevents me from submitting the form.

I figured that there are no more fields that we need to fill out on the CONFIRM step and all of the validation should have happened in the previous step so we can assume that the data that we have on confirm step is correct and as a workaround we can skip the validation.

Reply:
Ok, I supposed this was the case, thanks for explaining